src/Jackett.Common/Definitions/unleashthecartoons.yml

```diff
@@ -0,0 +1,141 @@
+---
+id: unleashthecartoons
+name: UnleashTheCartoons
+description: "UnleashTheCartoons is a Private Torrent Tracker for ANIMATED MOVIES / TV"
+language: en-US
+type: private
+encoding: UTF-8
+links:
+  - https://unleashthecartoons.world/
+
+caps:
+  categorymappings:
+    - {id: 1, cat: Movies, desc: "Movies"}
+    - {id: 2, cat: TV, desc: "TV Series"}
+    - {id: 3, cat: TV/Anime, desc: "Anime"}
+    - {id: 4, cat: Books, desc: "Ebooks"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep]
+    movie-search: [q]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: id
+    options:
+      id: created
+      seeders: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  path: account-login.php
+  method: post
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    returnto: /
+  error:
+    - selector: div.myFrame-caption:contains("Error")
+      message:
+        selector: div.myFrame-content
+    - selector: div.myFrame-caption:contains("Access denied")
+      message:
+        selector: div.myFrame-content
+  test:
+    path: /
+    selector: a[href="account-logout.php"]
+
+search:
+  paths:
+    # https://unleashthecartoons.world/torrents-search.php?search=&cat=0&incldead=1&freeleech=0&inclexternal=0&lang=0&sort=id&order=desc
+    - path: torrents-search.php
+  inputs:
+    search: "{{ .Keywords }}"
+    # does not support multi cats searching so defaulting to all.
+    cat: 0
+    # 0 active, 1 incldead, 2 onlydead
+    incldead: 1
+    freeleech: "{{ if .Config.freeleech }}2{{ else }}0{{ end }}"
+    # 0 Local/External, 1 Local Only, 2 External Only
+    inclexternal: 0
+    # 0 all, 1 Romanian, 2 English, 3 Romanian/English
+    lang: 0
+    sort: "{{ .Config.sort }}"
+    order: "{{ .Config.type }}"
+    # does not support multicategory filtering
+    # does not support imdbid searches, does not return imdb link in results
+
+  keywordsfilters:
+    - name: re_replace
+      args: ["\\.", " "]
+    - name: re_replace
+      args: ["\\s+", " "]
+    - name: re_replace
+      args: ["(\\w+)", "+$1"] # prepend + to each word
+
+  rows:
+    selector: table.ttable_headinner tr.t-row
+
+  fields:
+    category:
+      text: 1
+    category|noappend:
```

Comment:
we don't normally preload cats unless the site regularly has rows without a cat.
this is so that if the site adds a new cat we get to see what it is during the TEST and can keep the indexer up to date.
should the site regularly have missing cats from rows, we would then usually choose the cat **Other** as a default.
if the site does not have Other as a cat then its a matter of picking the most popular cat for the site.
does any of this apply here?